Index: src/main/java/ca/ucalgary/seng/p3/GameApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ca.ucalgary.seng.p3;\r\nimport javafx.application.Application;\r\nimport javafx.fxml.FXMLLoader;\r\nimport javafx.scene.Scene;\r\nimport javafx.scene.layout.AnchorPane;\r\nimport javafx.stage.Stage;\r\n\r\nimport java.io.IOException;\r\n\r\npublic class GameApplication extends Application{\r\n    @Override\r\n    public void start(Stage stage) throws IOException {\r\n        FXMLLoader fxmlLoader = new FXMLLoader(GameApplication.class.getResource(\"/connect4.fxml\"));\r\n        Scene scene = new Scene(fxmlLoader.load(), 900, 600);\r\n\r\n        stage.setTitle(\"Placeholder Title\");\r\n        stage.setScene(scene);\r\n        stage.show();\r\n    }\r\n\r\n    public static void main(String[] args) {\r\n        launch();\r\n    }\r\n}\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/ca/ucalgary/seng/p3/GameApplication.java b/src/main/java/ca/ucalgary/seng/p3/GameApplication.java
--- a/src/main/java/ca/ucalgary/seng/p3/GameApplication.java	(revision afd12a47df50d3473363739a7630a23475bd6b38)
+++ b/src/main/java/ca/ucalgary/seng/p3/GameApplication.java	(date 1742943595001)
@@ -10,7 +10,8 @@
 public class GameApplication extends Application{
     @Override
     public void start(Stage stage) throws IOException {
-        FXMLLoader fxmlLoader = new FXMLLoader(GameApplication.class.getResource("/connect4.fxml"));
+        FXMLLoader fxmlLoader = new FXMLLoader(GameApplication.class.getResource("/landing.fxml"));
+        fxmlLoader.setRoot(new AnchorPane());
         Scene scene = new Scene(fxmlLoader.load(), 900, 600);
 
         stage.setTitle("Placeholder Title");
